src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -44,10 +46,14 @@ def load_arguments(self, _):
         c.argument('admin_enabled', help='Indicates whether the admin user is enabled', choices=['true', 'false'])
 
     with self.argument_context('acr repository delete') as c:
-        c.argument('manifest', nargs='?', required=False, const='', default=None, help='The sha256 based digest of manifest to delete')
+        c.argument('manifest', nargs='?', required=False, const='', default=None, help=argparse.SUPPRESS)
         c.argument('yes', options_list=['--yes', '-y'], action='store_true', help='Do not prompt for confirmation')
-        c.argument('repository', help='The name of the repository to delete.')
-        c.argument('tag', help='The name of tag to delete')
+        c.argument('repository', help="The name of the repository to delete. For example, 'hello-world'")
+        c.argument('tag', help=argparse.SUPPRESS)
```

Comment:
Again, this still allows `--tag` to work, which may or may not be the desired behavior. If you DO intend to still allow this, consider adding a validator that exposes a deprecation warning when these params are used, especially if you intended to remove this via breaking change in the future.

Reply:
Yes we still want to support `--tag`. I am not putting a deprecation message here because different combinations of parameters (`--repository`, `--tag`, and `--manifest`) are refactored to different new commands and as you also commented below we are printing deprecation messages on each usage.